docs/api/python/gluon/contrib.md

```diff
@@ -16,10 +16,6 @@ many useful experimental APIs for new features.
 This is a place for the community to try out the new features,
 so that feature contributors can receive feedback.
 
-```eval_rst
-.. warning:: This package contains experimental APIs and may change in the near future.
```

Comment:
It's probably a good idea to keep the expectation on the contrib package to be experimental.